backend/de.metas.shipper.gateway.commons/src/main/java/de/metas/shipper/gateway/commons/model/ShipmentOrderRepository.java

```diff
@@ -68,29 +70,29 @@ public DeliveryOrder getById(@NonNull final DeliveryOrderId deliveryOrderId)
 	{
 		final I_Carrier_ShipmentOrder shipmentOrder = InterfaceWrapperHelper.load(deliveryOrderId, I_Carrier_ShipmentOrder.class);
 
-		final ImmutableListMultimap<ShipmentOrderParcelId, ShipmentOrderItem> parcelIdToItems = queryBL.createQueryBuilder(I_Carrier_ShipmentOrder_Parcel.class)
+		final ImmutableListMultimap<DeliveryOrderLineId, DeliveryOrderItem> parcelIdToItems = queryBL.createQueryBuilder(I_Carrier_ShipmentOrder_Parcel.class)
 				.addEqualsFilter(I_Carrier_ShipmentOrder_Parcel.COLUMNNAME_Carrier_ShipmentOrder_ID, deliveryOrderId)
 				.andCollectChildren(I_Carrier_ShipmentOrder_Item.COLUMNNAME_Carrier_ShipmentOrder_Parcel_ID, I_Carrier_ShipmentOrder_Item.class)
 				.create()
 				.stream()
 				// FIXME: not sure we have a link from parcel to item
-				.collect(ImmutableListMultimap.toImmutableListMultimap(item -> ShipmentOrderParcelId.ofRepoId(item.getCarrier_ShipmentOrder_Parcel_ID()), this::itemFromPO));
+				.collect(ImmutableListMultimap.toImmutableListMultimap(item -> DeliveryOrderLineId.ofRepoId(item.getCarrier_ShipmentOrder_Parcel_ID()), this::itemFromPO));
```

Comment:
why not renaming 
* DeliveryOrderLineId to DeliveryOrderParcelId
* DeliveryOrderLine to DeliveryOrderParcel
?